docs/modules/secret-operator/pages/cert-manager.adoc

```diff
@@ -37,46 +39,28 @@ include::example$cert-manager/issuer.yaml[]
 [#secretclass]
 == Creating a SecretClass
 
-The Stackable Secret Operator needs to know how to find the certificates created by Cert-Manager. We do this by creating
-a xref:secretclass.adoc[] using the xref:secretclass.adoc#backend-k8ssearch[`k8sSearch` backend], which can find arbitrary
-Kubernetes Secret objects that have the correct labels.
+The Stackable Secret Operator needs to know how to request the certificates from Cert-Manager. We do this by creating
+a xref:secretclass.adoc[] using the xref:secretclass.adoc#backend-certmanager[`experimentalCertManager` backend].
 
 [source,yaml]
 ----
 include::example$cert-manager/secretclass.yaml[]
 ----
 <1> Both certificates and Pods will reference this name, to ensure that the correct certificates are found
-<2> This informs the Secret Operator that certificates will be found in the same namespace as the Pod using it
-
-[#certificate]
-== Requesting a certificate
-
-You can now use Cert-Manager to provision your first certificate. Use labels to inform the Stackable Secret Operator
-about which xref:scope.adoc[scopes] the certificate fulfills. Which scopes must be provisioned is going to depend
-on the design of the workload. This guide assumes the xref:scope.adoc#service[service] scope.
-
-[source,yaml]
-----
-include::example$cert-manager/certificate.yaml[]
-----
-<1> The Certificate name is irrelevant for the Stackable Secret Operator's, but must be unique (within the Namespace)
-<2> The Secret name must also be unique within the Namespace
-<3> This tells the Stackable Secret Operator that this secret corresponds to the SecretClass created xref:#secretclass[before]
-<4> This secret fulfils the xref:scope.adoc#service[service] scope for `my-app`
-<5> The list of DNS names that this certificate should apply to.
-<6> The Cert-Manager Issuer that should sign this certificate, as created xref:#issuer[before]
+<2> This guide uses a namespaced Issuer, rather than a cluster-scoped ClusterIssuer
+<3> The Cert-Manager Issuer that should sign these certificates, as created xref:#issuer[before]
 
 [#pod]
 == Using the certificate
 
-Finally, we can create and expose a Pod that consumes the certificate!
+Finally, we can create and expose a Pod that requests and uses the certificate!
 
 [source,yaml]
 ----
 include::example$cert-manager/pod.yaml[]
 ----
 <1> A secret xref:volume.adoc[volume] is created, where the certificate will be exposed to the app
 <2> The volume references the SecretClass defined xref:#secretclass[before]
-<3> The app is designated the scope xref:scope#service[`service=my-app`], matching the xref:#certificate[certificate's scope]
+<3> The app requires the certificate to be valid for the scopes xref:scope.adoc#node[`node`] and xref:scope.adoc#service[`service=my-app`]
```

Comment:
It's an arbitrary example. Moving forward, ideally everything would use the listener scope and not have to care about manually assigning scopes at all.

Reply:
Right. It just sounded like it was required because of Cert-Manager, so I thought some clarity might be helpful.

Feel free to resolve this conversation if you think it is fine.